src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/ErrorFmt.cs

```diff
@@ -76,35 +76,35 @@ public ErrArg()
         }
         public ErrArg(int n)
         {
-            this.eak = ErrArgKind.Int;
-            this.eaf = ErrArgFlags.None;
```

Comment:
We should only remove the ```this.``` at the same time we rename fields to begin with underscore.  Otherwise, based on expectations set elsewhere in the repo, it's too easy to misinterpret these as locals, or at least to be confused by it.